flocker/common/test/test_interface.py

```diff
@@ -0,0 +1,133 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+Tests for ``flocker.common.interface_decorator``.
+"""
+
+from twisted.trial.unittest import SynchronousTestCase
+
+from eliot.testing import (
+    assertHasMessage, capture_logging
+)
+from eliot import Field, MessageType
+
+from zope.interface import Interface, implementer
+
+from ..common import interface_decorator
```

Comment:
Don't need the `common` here...just `from .. import interface_decorator` i think.

```
[ERROR]
Traceback (most recent call last):
  File "/srv/buildslave/flocker-centos-7/venv/lib/python2.7/site-packages/twisted/trial/runner.py", line 497, in loadPackage
    module = modinfo.load()
  File "/srv/buildslave/flocker-centos-7/venv/lib/python2.7/site-packages/twisted/python/modules.py", line 383, in load
    return self.pathEntry.pythonPath.moduleLoader(self.name)
  File "/srv/buildslave/flocker-centos-7/venv/lib/python2.7/site-packages/twisted/python/reflect.py", line 303, in namedAny
    topLevelPackage = _importAndCheckStack(trialname)
  File "/srv/buildslave/flocker-centos-7/venv/lib/python2.7/site-packages/twisted/python/reflect.py", line 250, in _importAndCheckStack
    reraise(excValue, excTraceback)
  File "/srv/buildslave/flocker-centos-7/venv/lib/python2.7/site-packages/flocker/common/test/test_interface.py", line 16, in <module>
    from ..common import interface_decorator
exceptions.ImportError: No module named common

flocker.common.test.test_interface
```